rpm/tarantool.spec

```diff
@@ -147,7 +153,9 @@ C and Lua/C modules.
 
 %build
 # RHBZ #1301720: SYSCONFDIR an LOCALSTATEDIR must be specified explicitly
-%cmake . -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+# since Fedora 33 LTO flags were set in CC/LD flags by default
```

Comment:
> It is better to place this comment right before the flag.

I believe this change will separate the multiline cmake command by parts and break it. I've tried to do this in scope of [this](https://lists.tarantool.org/tarantool-patches/cover.1611149713.git.imun@tarantool.org/T/#t) series but failed.

Reply:
@igormunkin Thanks for the clarification.

@avtikhon So, please, fix punctuation. I would also suggest to separate different comments with an 'empty' line:

```
# Here I described a point A.
#
# Here I describe another point, B.
```

Reply:
Not possible to set the commit just before the flag, because of syntax. Anyway changed the message to point the flags:
```
-# since Fedora 33 LTO flags were set in CC/LD flags by default
-%cmake -S . -B . \
+# Must specified explicitly since Fedora 33:
+# 1. -B .
+#    because for now binary path by default value like:
+#      '-B x86_64-redhat-linux-gnu'
+# 2. -DENABLE_LTO=ON
+#    because for now LTO flags are set in CC/LD flags by default:
+#      '-flto=auto -ffat-lto-objects'
+%cmake -B . \
```

Reply:
Now it is here.